src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -555,6 +557,19 @@ private function createAuthenticationListeners(ContainerBuilder $container, stri
             }
         }
 
+        if (!$entryPoints) {
+            // we can be sure the authenticator system is enabled
+            if (null !== $defaultEntryPoint) {
+                return $entryPoints[$defaultEntryPoint] ?? $defaultEntryPoint;
+            }
+
+            if (1 === \count($entryPoints)) {
+                return current($entryPoints);
+            }
+
+            throw new InvalidConfigurationException(sprintf('Because you have multiple authenticators in firewall "%s", you need to set the "entry_point" key to one of your authenticators (%s) or a service ID implementing "%s".', $id, implode(', ', $entryPoints), AuthenticationEntryPointInterface::class));
```

Comment:
Throwing an exception looks like a BC break, right? We should probably raise a deprecation instead and add the exception in 6.0, no?

Reply:
`$entryPoints` is only filled with entry points when you're using the new experimental system. For the current security system, entry points are still as magic as before.

I think it's OK to say "if you're using the new system, you have to configure the entry point if there is more than one authenticator".

Reply:
Nvm, I see that this causes the builds to fail. So probably I did something wrong, let me check.

Reply:
Ah, had to inverse the if condition :facepalm: